examples/interaction_energy/sdk/run.py

```diff
@@ -0,0 +1,142 @@
+import os
+
+from promethium_sdk.client import PromethiumClient
+from promethium_sdk.models import (
+    CreateInteractionEnergyCalculationWorkflowRequest,
+)
+from promethium_sdk.utils import base64encode
+
+foldername = "output"
+gpu_type = os.getenv("PM_GPU_TYPE", "a100")
+
+if not os.path.exists(foldername):
+    os.makedirs(foldername)
+
+moleculeA_adenine = base64encode(
+"""
```

Comment:
Let's make these properly formatted XYZ files i.e., 2-line header with atom count. While this "lazy" format is accepted, it's preferable to use the proper format

Reply:
Done!